### DISCUSSION TITLE
2023년 04월 20일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#43] [General]
# 2023년 04월 20일

LOG 네오, 에밀, 망고, 토리, 포이, 여우, 주노와 함께 점심을 먹었다. 원래 토도로끼에 갈 생각이었는데 사람들이 너무 많아서 네오가 아는 곳으로 제육을 먹으러 갔다. 네오와 같이 앉아서 이야기를 했는데 영화에 대한 이야기를 하다가 종교에 대한…

https://cyma-s.github.io/2023-04-20/

**Comment by Choi-JJunho** [comment]
given 구절에 대해서 BDD Mockito도 같이 정리해주세요

**Reply by Cyma-s** [reply]
오늘 시간 남으면 글 써보겠습니다 ~

**Comment by kevstevie** [comment]
>mock으로 테스트하는 게 정말 의미가 있을까?

제가 알려드렸잖아요

**Reply by Cyma-s** [reply]
컨트롤러의 정상 동작을 테스트 하기 위해 필요하다
- 헤이주드
